app-dicts/myspell-es/myspell-es-2.6.ebuild

```diff
@@ -18,24 +17,22 @@ MYSPELL_THES=(
 	"th_es_v2.idx"
 )
 
-SRC_URI=""
 for i in "${MUTATIONS[@]}"; do
 	MYSPELL_DICT+=(
 		"${i}.dic"
 		"${i}.aff"
 	)
-	SRC_URI+=" https://github.com/sbosio/rla-es/releases/download/v${PV}/${i}.oxt -> ${i}-${PV}.oxt"
+	SRC_URI+=" https://github.com/sbosio/rla-es/releases/download/v${PV}/${i}.oxt -> ${i}-${PV}.oxt" # leave the space or it will fails to download
```

Comment:
Self-explanatory with `SRC_URI+=`.

Reply:
I didn't know about that thing, so I left a comment if another n00b like me will se it